mungegithub/mungers/flake-manager.go

```diff
@@ -127,6 +127,11 @@ func (p *FlakeManager) isIndividualFlake(f cache.Flake) bool {
 	return true
 }
 
+// makeGubernatorLink returns a URL to view the build results in a GCS path.
+func makeGubernatorLink(id string) string {
+	return "https://k8s-gubernator.appspot.com/build" + id
+}
```

Comment:
Why is this an improvement over the direct link? It doesn't offer any more features, and it loads very slowly?